db/migration/20130108000008_create_tags.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class CreateTags < ActiveRecord::Migration[4.2]
+  def change
+    unless table_exists?(:tags)
+      create_table :tags do |t|
+        t.string :name, null: false
+
+        t.timestamps
+      end
+    end
+
+    if table_exists?(:taggings)
+      change_table :taggings do |t|
+        t.remove :tag_context
```

Comment:
As we don't use dm-tags, we don't need tag_context any longer. If we didn't drop this column, existing users would always fail saving entry.